test/DynamoCoreWpfTests/ConverterViewModelTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Controls;
+using CoreNodeModels;
+using CoreNodeModels.Input;
+using CoreNodeModelsWpf.Controls;
+using Dynamo.Tests;
+using Dynamo.Utilities;
+using DynamoCoreWpfTests;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace CoreNodeModelsWpf
+{
+    [TestFixture]
+    public class ConverterViewModelTests : DynamoTestUIBase
+    {
+        private AssemblyHelper assemblyHelper;
+
+        public override void Open(string path)
+        {
+            base.Open(path);
+
+            DispatcherUtil.DoEvents();
+        }
+
+        public override void Run()
+        {
+            base.Run();
+
+            DispatcherUtil.DoEvents();
+        }
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            SetupTests();
+            libraries.Add("VMDataBridge.dll");
+            libraries.Add("DSCoreNodes.dll");
+            libraries.Add("DynamoConversions.dll");
+            libraries.Add("DynamoUnits.dll");
+            libraries.Add("CoreNodeModelsWpf.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
+
+        public void SetupTests()
+        {
+            var assemblyPath = Assembly.GetExecutingAssembly().Location;
+            var moduleRootFolder = Path.GetDirectoryName(assemblyPath);
+
+            var resolutionPaths = new[]
+            {
+                // The CoreNodeModelsWpf.dll needed is under "nodes" folder.
+                Path.Combine(moduleRootFolder, "nodes")
```

Comment:
I think this test fixture should be using the `Setup` class test fixture that already adds this resolution path so this should not be needed again here. The `Setup` class is defined in `Setup.cs` in the same `DynamoCoreWpfTests` assembly.

Reply:
@aparajit-pratap Reminded me that we are already doing these in the base class